stable/pipelines/values.yaml

```diff
@@ -131,7 +131,19 @@ pipelines:
       ## Whitelist IPs allowed to LoadBalancer type services
       ## Example: loadBalancerSourceRanges={82.82.190.51/32,141.141.8.8/32}
       loadBalancerSourceRanges: []
-
+    livenessProbe:
```

Comment:
Also, can we have switch to enable/disable to  livenessProbe or readinessProbe , please refer [artifactory chart](https://github.com/jfrog/charts/blob/master/stable/artifactory/values.yaml#L511) for an example